Documentation/Functions/Select.rst

```diff
@@ -154,7 +154,18 @@ pidInList
       Allows to disable the :sql:`pid` constraint completely. Requirements: 
       :ts:`uidInList` *must* be set or the table *must* have the prefix 
       "static\_\*".
+      
+   .. note::
+      Check the doktype of your backend page. If you are trying to fetch records from
+      e.g. a folder, the :php:`$cObj->checkPid_badDoktypeList` method will insert the
```

Comment:
by folder you mean a page with type sys_folder?

Reply:
Yes, I updated the word to be more specific, thank you.